packages/adblocker-extended-selectors/rollup.config.ts

```diff
@@ -28,22 +32,37 @@ export default [
     ],
   },
   {
-    input: './dist/src/adblocker.js',
+    input: './adblocker.ts',
     output: [
       {
-        dir: './dist/esm',
+        dir: './dist',
         format: 'esm',
         preserveModules: true,
         entryFileNames: '[name].js',
         sourcemap: true,
       },
       {
-        dir: './dist/cjs',
+        dir: './dist',
         format: 'cjs',
         preserveModules: true,
         entryFileNames: '[name].cjs',
```

Comment:
I'm afraid of not being backward-compatible with old versions of Node.JS by using `cjs` extension instead of `js` for output files. I'd like to suggest using `mjs` for ESM and `js` for CJS. Only we need to take care is emitting `d.cts` properly.

Reply:
I think it's best to split the directory and use `js` extension for both ESM and CJS. For ESM, I think `mjs` extension is all fine.

Reply:
Checked that this is not a problem if we specify the full file extension.